content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -293,6 +293,120 @@ Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-pl
+
+### Services
+
+The IPv4/IPv6 dual stack enhancement includes the capability for Services to be created with either an IPv4 or an IPv6 address. The `Service.Spec.IPFamily` field has been added to Services to enable users to designate which address family to use for the Service IP. The `Service.Spec.IPFamily` is an immutable field. The default value of this field is the address family of the first service cluster IP range configured via the `--service-cluster-ip-range` flag on kube-controller manager.
+
+Possible values of this field are as follows:
+
+   * IPv4: api-server will assign an IP from a `service-cluster-ip-range` that is `ipv4`
+   * IPv6: api-server will assign an IP from a `service-cluster-ip-range` that is `ipv6`
+
+The following Service specification includes the `ipFamily` field. Kubernetes will assign an IPv6 address (also known as a "cluster IP") from the configured `service-cluster-ip-range` to this Service.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  ipFamily: IPv6
+  selector:
+    app: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+```
+
+For comparison, the following Service specification will be assigned an IPV4 address (also known as a "cluster IP") from the configured `service-cluster-ip-range` to this Service.
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  ipFamily: IPv4
+  selector:
+    app: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+```
+
+#### Type LoadBalancer
+
+On cloud providers which support IPv6 enabled external load balancers, setting the `type` field to `LoadBalancer` provisions a cloud load balancer for your Service.
+
+### Egress Traffic
+
+In enable IPv6 Pod routing to off-cluster destinations (eg. the Internet) the IPv6 Pod interface must be masqueraded before egressing the node. The [ip-masq-agent](https://github.com/kubernetes-incubator/ip-masq-agent) is dual stack aware and may be used to manage IP masquerading on dual stack enabled clusters.
+
+### Provisioning a dual stack enabled cluster
+
+The following open source tooling supports the delivery of a dual stack enabled Kubernetes cluster
+
+   * [AKS Engine](https://github.com/Azure/aks-engine/tree/master/examples/dualstack)
+
+### Ecosystem tooling
+
+The dual stack enhancement is supported in Kubernetes core. Ecosystem tooling and projects including CoreDNS and CNI plugins may choose to implement dual stack capabilities. 
+
+### Known Issues
+
+   * IPv6 network block assignment is using the default IPv4 cidr block size (/24)
+   * Kubenet forces IPv4,IPv6 positional reporting of IPs (--cluster-cidr)
+   * Dual stack will not function if `EndpointSlice` enhancement is enabled.
+
+### What's Next
+
+Details on the implementation of this feature may be found in the [IPv4/IPv6 dual stack](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md) KEP.
```

Comment:
I recommend omitting this. The What's Next section is for readers viewing whole page, not just the subset who want to learn about IPv6 / dualstack.